.github/workflows/build.yml

```diff
@@ -10,6 +23,9 @@ jobs:
     name: Build
```

Comment:
Since we're using this as part of our release step, let's create a separate job for building the demo. While the demo isn't needed for releases, we do want to ensure it's buildable on each PR.

This also means that the strategy matrix in this job can be removed.

Reply:
created separate job